force-app/main/default/classes/TriggerFactory.cls

```diff
@@ -0,0 +1,41 @@
+public class TriggerFactory {
+	public class TriggerException extends Exception {
+	}
+	public static void execute(SObjectType sobjectType) {
+		ITriggerHandler handler = createHandler(sobjectType);
+		handler.disableTrigger();
+		handler.enableTrigger();
+		if (!handler.isTriggerEnabled()) {
+			return;
+		}
+		handleOperation(handler);
+	}
+
+	private static void handleOperation(ITriggerHandler handler) {
+		if (Trigger.isUpdate) {
+			if (Trigger.isAfter) {
+				handler.afterUpdate(Trigger.newMap, Trigger.oldMap, Trigger.new, Trigger.old);
+			} else if (Trigger.isBefore) {
+				handler.beforeUpdate(Trigger.newMap, Trigger.oldMap, Trigger.new, Trigger.old);
+			}
+		}
+	}
+
+	private static ITriggerHandler createHandler(SObjectType sobjectType) {
+		String originalTypeName = sobjectType.getDescribe().getName();
+		String handlerTypeName = null;
+		if (originalTypeName.toLowerCase().endsWith('__c')) {
+			Integer index = originalTypeName.toLowerCase().indexOf('__c');
+			handlerTypeName = originalTypeName.substring(0, index) + 'TriggerHandler';
+		} else {
+			handlerTypeName = originalTypeName + 'TriggerHandler';
+		}
+		handlerTypeName = handlerTypeName.replaceAll('_', '');
+		Type obType = Type.forName(handlerTypeName);
```

Comment:
Great idea, but maybe it's better to use just some map in this class like Map<SObjectType, Type>(where class would be kept like Opportunity.SobjectType =>OpportunityTriggerHandler.class) and take value from it.
Because it can be that class name differs from what we expected